core/src/main/kotlin/org/evomaster/core/output/clustering/metrics/DistanceMetricErrorText.kt

```diff
@@ -18,9 +18,12 @@ import javax.ws.rs.core.MediaType
  *
  */
 
-class DistanceMetricErrorText : DistanceMetric<RestCallResult>() {
+class DistanceMetricErrorText(
+        epsilon: Double = 0.6
+) : DistanceMetric<RestCallResult>() {
     private val name = "ErrorText"
-    private val recommendedEpsilon = 0.6
+    private val recommendedEpsilon = if (epsilon in 0.0..1.0) epsilon
```

Comment:
if someone provides a wrong input, instead of silently change it into a 0.6, should rather throw an `IllegalArgumentException`.
anyway, constraints in EMConfig will already handle it